packages/client/src/products/__fixtures__/getListing.fixtures.ts

```diff
@@ -1,37 +1,18 @@
-import join from 'proper-url-join';
-import moxios from 'moxios';
-import type { Listing, ListingQuery } from '../types';
+import { rest, RestHandler } from 'msw';
+import type { Listing } from '../types';
+
+const path = '/api/commerce/v1/listing/:gender/:category';
```

Comment:
I agree, and that's what I had initially, but unfortunately our mock slug is something like `/woman/clothing` and `msw` does not catch it as `.../list/:slug`. Any suggestions? I could either change the mock slug or use more "generic" parameters.

Reply:
Ahhhhh yeah, it makes sense 😕. What about using the `path wildcard` as described [here](https://mswjs.io/docs/basics/request-matching#request-url)? We'd have to change our path to `/api/commerce/v1/listing/*` but that wouldn't be wrong since every listing request have that "base" and the rest can be a lot of stuff that we can't predict. Hopefully, it works 🤗

Reply:
It worked! Thank you!!!